heka/sandbox/filters/telemetry_s3output_monitors.lua

```diff
@@ -54,10 +56,16 @@ function process_message ()
             if v.Name ~= "TelemetryErrorsOutput" then
                 n = v.ProcessMessageCount.value
                 if type(n) == "number" then
-                    if n == p.last_pmc then -- no message in the Dashboard ticker_interval
-                        p.msg = string.format("%s ProcessMessageCount has stalled at %d", v.Name, n)
+                    if n == p.last_pmc then
+                        if p.last_update + 60 * 1e9 < ts then
```

Comment:
If we ran multiple instances with different settings then a config option (specified in puppet) would be my choice otherwise I would rather modify the source.